cockroachcloud/ccloud-get-started.md

```diff
@@ -317,6 +326,35 @@ warning: server version older than client! proceed with caution; some features m
 user@free-tier7.gcp-us-central1.crdb.io:26257/defaultdb>
 ~~~
 
+### Connect to your cluster using SSO
+
+Use the `--sso` flag to connect to your cluster using [single sign-on (SSO) authentication](cloud-sso-sql.html). 
```

Comment:
Can we add some additional context here, something like "this will allow you to start a SQL shell without using a password for SQL authentication"

Reply:
Done.